app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  # Method that returns the 3 most recent posts for a given user.
+
+  def find_posts
+    Post.where(author: self).order(updated_at: :desc).first(3)
+  end
```

Comment:
- [ ] Instead of using "updated_at" ,kindly use "created_at".
```
def recent_post
  posts.order('created_at Desc').limit(3)
end
```

Reply:
Updated !!